data/gui/screens/help1.stkgui

```diff
@@ -54,28 +54,28 @@
                     <spacer width="25" height="25"/>
                     <bubble proportion="3" height="100%" word_wrap="true"
                             I18N="In the help menu"
-                            text="Collecting nitro allows you to get speed boosts whenever you wish by pressing the appropriate key or button. You can see your current level of nitro in the gauge at the bottom-right of the race screen."/>
-                    </div>
-
-                <div width="100%" proportion="4" layout="horizontal-row">
-                    <icon align="center" width="64" height="64" icon="gui/icons/gui_lock.png"/>
+                            text="Collecting nitro allows you to use small speed boosts by pressing the assigned key or button*. You can see your current level of nitro in the gauge at the bottom-right of the race screen."/>
+                </div>
+                
+                <div width="100%" proportion="7" layout="horizontal-row">
+                    <icon align="center" width="64" height="64" icon="gui/icons/android/drift.png"/>
                     <spacer width="25" height="25"/>
                     <bubble proportion="2" height="100%" word_wrap="true"
                             I18N="In the help menu"
-                            text="If you see a button with a lock like this one, you need to complete a challenge to unlock it."/>
+                            text="You can skid by pressing the assigned key or button*. Successive short skids help to take sharp turns; while medium to long skids will give you a small or medium boost, depending on the length of the skid. You can't stop turning while skidding, so orient your kart carefully before!"/>
```

Comment:
There is a reason my formulation for the skid help text was not the one you suggested initially. Yours was too long. This new modification suffers from the same flaw.

What's wrong with the current text ?